src/pages/Login/LoginForm/LoginForm.js

```diff
@@ -1,18 +1,65 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
 import CommonForm from '../../../component/Form/CommonForm';
 import CommonInput from '../../../component/Input/CommonInput';
 import { LOGIN_INPUT_LIST } from './LOGIN_INPUT_LIST';
+import { LOGIN_API } from '../../../config';
 
 import './LoginForm.scss';
 
 class LoginForm extends React.Component {
+  loginSubmit = e => {
+    e.preventDefault();
+    const { history } = this.props;
+    const { loginId, loginPw } = this.props.userInfo;
+
+    fetch(LOGIN_API, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: loginId,
+        password: loginPw,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.token) {
+          history.push(`/`);
+          localStorage.setItem(`loginToken`, data.token);
+        } else {
+          alert(`ID, 비밀번호를 확인해주세요`);
+        }
+      });
+  };
+
   render() {
-    const loginInpuList = LOGIN_INPUT_LIST.map(elem => {
-      const { type, name, placeholder } = elem;
-      return <CommonInput type={type} name={name} placeholder={placeholder} />;
-    });
+    const loginInpuList = LOGIN_INPUT_LIST.map(
+      ({ id, type, name, placeholder }) => {
+        const { setLoginInfo } = this.props;
+
+        return (
+          <CommonInput
+            key={id}
+            type={type}
+            name={name}
+            placeholder={placeholder}
+            handleOnChange={e => {
+              setLoginInfo(name, e.target.value);
+            }}
+          />
+        );
+      }
+    );
+
+    const { loginSubmit } = this;
+    const { userInfo } = this.props;
+
     return (
-      <CommonForm cases="로그인">
+      <CommonForm
+        type="signIn"
+        cases="로그인"
+        userInfo={userInfo}
+        handleSubmit={e => loginSubmit(e)}
```

Comment:
```suggestion
        handleSubmit={loginSubmit}
```
굳이 인라인 함수를 다시 선언할 필요 없어보입니다.